java/arcs/android/devtools/DevWebServerImpl.kt

```diff
@@ -25,6 +25,7 @@ object DevWebServerImpl : DevWebServer, NanoWSD("localhost", 33317) {
 
     private val wsdSockets = mutableSetOf<WsdSocket>()
     private val log = TaggedLog { "DevWebSocket" }
+    private val onOpenSocketCallbacks = mutableListOf<() -> Unit>()
```

Comment:
Super minor nit (optional really): I think `mutableSetOf` would be ever so slightly faster here for the `removeOnOpenWebsocketCallback` case - removing from a HashSet is O(1), whereas removing from a List is O(n).

The reason why it's optional in my mind is that I doubt we'll ever have enough listeners for it to make a material difference in performance.